### PR TITLE
Integration tests: wait for green at startup

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

**Comment by Mpdreamz** [comment]
The test failures are fixed in #4364 

https://dev.azure.com/azure3/elasticsearch-net/_build/results?buildId=2964&view=ms.vss-test-web.build-test-results-tab

File: tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs [hunk 1]
```diff
@@ -23,6 +24,15 @@ public abstract class ClientTestClusterBase : XunitClusterBase<ClientTestCluster
 		public IElasticClient Client => this.GetOrAddClient(s => ConnectionSettings(s.ApplyDomainSettings()));
 
 		protected virtual ConnectionSettings ConnectionSettings(ConnectionSettings s) => s;
+
+		protected sealed override void SeedCluster()
+		{
+			Client.Cluster.Health(new ClusterHealthRequest { WaitForStatus = WaitForStatus.Green });
```
Comment (russcam) [inline tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs]: should this check the response?

File: tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs [hunk 2]
```diff
@@ -23,6 +24,15 @@ public abstract class ClientTestClusterBase : XunitClusterBase<ClientTestCluster
...
+			SeedNode();
+			Client.Cluster.Health(new ClusterHealthRequest { WaitForStatus = WaitForStatus.Green });
```
Comment (russcam) [inline tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs]: should this check the response?

File: tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs (hunk 1 above)
Comment (Mpdreamz) [reply tests/Tests.Core/ManagedElasticsearch/Clusters/ClientTestClusterBase.cs]: I'm okay to fare blind here, let's see how it behaves first before deciding.

**Pull request merged**